### ISSUE TITLE
Document functions

### ISSUE THREAD

**Issue by gedankenstuecke** [OpenHumans/open-humans-api#5]
Besides the CLI tools the package contains functions that can be used to interface with the OH API from Python itself. These are not documented so far.

**Comment by VineetaMotilal** [comment]
@gedankenstuecke this is available right? I can take this up as the project!

**Comment by gedankenstuecke** [comment]
@VineetaMotilal absolutely, please feel free to go ahead with that!

**Comment by VineetaMotilal** [comment]
@gedankenstuecke Sir, can you mention the functions which need to be documented so that I can go ahead with it.

**Comment by gedankenstuecke** [comment]
@VineetaMotilal Basically all the functions in `ohapi/api.py` are not having any documentation. It would be great if all of them could be documented, for now maybe in a `USAGE.md` in the root-folder of the repository. Later on we might want to do a readthedocs page or something else for it, but a single file would be a good start.

Things to document would be the different parameters needed for the function calls and what the functions are doing.

**Comment by DartingMelody** [comment]
@VineetaMotilal, are you still working on this issue? Actually, I want to work on it, so let me know. Thanks :)

**Comment by VineetaMotilal** [comment]
@DartingMelody please go ahead.

**Comment by kritisingh1** [comment]
@DartingMelody since there are so many functions in ohapi/api.py, I think it would be better if we work together here too :)
I am taking up the oauth2_auth_url and oauth2_token_exchange for now. Thanks :)

**Comment by DartingMelody** [comment]
Yeah, sure! It would be great. Feel free to work on them. I will be working on rest of the functions.
